src/server/helpers/utils.js

```diff
@@ -52,13 +72,36 @@ function getEntityModelByType(type) {
 	return entityModels[type];
 }
 
+/**
+ * Regular expression for valid BookBrainz IDs
```

Comment:
BBID is being used a lot in this project so I'll avoid changing any potentially breaking identifiers for now.